dotnet/src/webdriver/Internal/Logging/LogLevel.cs

```diff
@@ -0,0 +1,12 @@
+namespace OpenQA.Selenium.Internal.Logging
+{
+    public enum Level
```

Comment:
Enum named `Level`, but file is `LogLevel.cs`. I understand that we already have `LogLevel` in the library and we have a name conflict. I propose to name this enum as `LogEventLevel`. For example, Serilog uses such name for analogous log level enum.

Reply:
Renamed to `LogEventLevel`